### PR TITLE
Feature flask integration

### PR DESCRIPTION
**Pull request by kimaya1008**
Created localhost file with flask integration.
Added modifications to spellCheck.py file

### CODE REVIEW

File: Flask-Integration/get-localhost.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from flask import Flask, jsonify, request
+from spellCheck import *
+
+#export FLASK_APP=test1.py
```
Comment (rjuliang) [inline Flask-Integration/get-localhost.py:4]: Can we change the name of the files in this command?

File: Flask-Integration/get-localhost.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#flask run
+
+app = Flask(__name__)
+
+@app.route('/process')
+def dataGet():
```
Comment (rjuliang) [inline Flask-Integration/get-localhost.py:10]: Can we update the name of this function to processSpellCheck or similar?

File: Flask-Integration/get-localhost.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    phrase = request.args.get('phrase', None)
+    language = request.args.get('lng', None)
+
+    if phrase and language:
+        print('phrase: '+phrase)
+        errorList = checkPhrase(phrase)
+        print('After error list')
```
Comment (rjuliang) [inline Flask-Integration/get-localhost.py:17]: It looks like we may not need this print statement, can we delete it?

**Review by rjuliang — CHANGES_REQUESTED** [review]
Everything looks pretty good, however, I would like some little changes

File: Flask-Integration/get-localhost.py (hunk 1 above)
Comment (kimaya1008) [reply Flask-Integration/get-localhost.py]: Done, changed to file name

File: Flask-Integration/get-localhost.py (hunk 2 above)
Comment (kimaya1008) [reply Flask-Integration/get-localhost.py]: Changed the name to processSpellCheck

File: Flask-Integration/get-localhost.py (hunk 3 above)
Comment (kimaya1008) [reply Flask-Integration/get-localhost.py]: Removed print statement

**Review by rjuliang — APPROVED** [review]
Thanks for making the changes Kimaya :) 
Everything looks good.

File: Flask-Integration/spellCheck.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+def words(text): return re.findall(r'\w+', text)
+
+WORDS = Counter(words(open('bigText.txt').read()))
+
+def P(word, N=sum(WORDS.values())): 
+    "Probability of `word`."
+    return WORDS[word] / N
+
+#def correction(word):
```
Comment (Pream9) [inline Flask-Integration/spellCheck.py:22]: Why is this in the comment line?

**Review by Pream9 — COMMENTED** [review]
Good work guys

**Pull request merged**